lib/host.rb

```diff
@@ -529,9 +529,9 @@ def delete(file, opts={})
       else
         exec_method = :exec
       end
-      public_send(exec_method, "rm #{r} -f -- #{file}", opts)
+      public_send(exec_method, "rm #{r} -f -- #{file}", **opts)
       opts[:quiet] = true
-      res = public_send(exec_method, "ls -d -- #{file}", opts)
+      res = public_send(exec_method, "ls -d -- #{file}", **opts)
```

Comment:
You sure we need these changes?

Reply:
@akostadinov yes.  https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/